src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -529,8 +532,220 @@ ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::Get
+        return [=](const Handle& handle, const boost::any& primitive_params) {
+            const auto invoke_params = boost::any_cast<conv::WrWInvokeParams>(primitive_params);
+            const auto& tensors      = invoke_params.tensors;
+            float total_time         = 0;
+
+            if(invoke_params.workSpaceSize < ws_sz)
+                MIOPEN_THROW("Not enough workspace for ConvWinograd3x3MultipassWrW");
+
+            for(const auto& kernel : kernels)
+            {
+                decltype(auto) cur_kernel = handle.Run(kernel);
+                const BuffInfo* d_buf     = nullptr;
+                const BuffInfo* o_buf     = nullptr;
+                Data_t buff_out_adr       = nullptr;
+                auto f_buf                = &weights_buff_info;
+                auto const_buff_in_adr    = tensors.x;
+                auto buff_in_adr          = invoke_params.workSpace;
+                bool const_input          = false;
+                float cur_time            = 0;
+                int flat_GroupCountMult   = 1;
+
+                size_t buff_in_addr_offset = 0, buff_out_addr_offset = 0;
+
+                if(cur_kernel.GetName() == GetSolverKernelNames(0)) // Input Transform
+                {
+                    d_buf               = &in_buff_info;
+                    o_buf               = &(wino_in.buff_info);
+                    const_buff_in_adr   = tensors.x;
+                    buff_out_adr        = invoke_params.workSpace;
+                    buff_in_addr_offset = wino_in_offset;
+                    const_input         = true;
+                    flat_GroupCountMult = GetGroupCountMult();
+                }
+                else if(cur_kernel.GetName() == GetSolverKernelNames(1)) // Filter Transform
+                {
+                    d_buf                = &weights_buff_info;
+                    o_buf                = &(wino_wei.buff_info);
+                    const_buff_in_adr    = tensors.dy;
+                    buff_out_adr         = invoke_params.workSpace;
+                    buff_out_addr_offset = wino_wei_offset;
+                    const_input          = true;
+                    flat_GroupCountMult  = GetGroupCountMult();
+                }
+                else // Output and GEMM
+                {
+                    int m = N, n = K, k = wino_in.buff_info.size.c;
+                    int lda = k, ldb = k, ldc = n;
+                    int batch_count       = wino_xform_h * wino_xform_w;
+                    long long int strideA = m * k * 1LL, strideB = k * n * 1LL,
+                                  strideC = m * n * 1LL;
+                    float alpha = 1., beta = 0.0;
+                    // clang-format off
+                    GemmDescriptor wino_gemm_desc{false,false,true,m,n,k,
+                        lda,ldb,ldc,batch_count,strideA,strideB,
+                                        strideC,alpha,beta,in_data_type};
+
+                    CallGemmStridedBatched(handle,
```

Comment:
CallGemmTimeMeasure is still not fixed, so it's better to use only CallGemmStridedBatched.